upgrade/1.2/resource_material/storage/cephadm-reference.md

```diff
@@ -1,17 +1,18 @@
 # CEPHADM
+
 `cephadm` is a new function introduced in Ceph Octopus 15. It allows for an easier method to install and manage Ceph nodes.
 
 ## Traditional Ceph commands
 
-On `ncn-s001`, `ncn-s002`, or `ncn-s003`:
+On any storage node:
```

Comment:
Are you sure this can be any storage node? My understanding is that some things only work on the 3 ceph mon nodes, which traditionally are the first 3 storage nodes.

Reply:
it is specifically calls out ncn-s001/2/3 then that is on purpose.
ceph cluster level commands can only be run on m001/2/3 and s001/2/3.  there are other ceph commands that can be run if it is a task specific to the storage node you are on.  example cephadm shell -- ceph-volume ......

Reply:
My page wasn't refreshed when I left my comment above.